ansible-scylla-node/tasks/repair.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Start repair
```

Comment:
Please, add an inline description explaining that this does exactly: it repairs primary token range belonging the node with a REST served from `scylla_api_address`.

Reply:
Done